open_humans/templates/pages/data-processing-activities.html

```diff
@@ -0,0 +1,206 @@
+{% extends 'base.html' %}
+
+{% block head_title %}GDPR{% endblock %}
+
+{% block main %}
+<div class="row">
+  <div class="col-lg-12">
+    <h2 class="page-header">Open Humans: Records of Personal Data Processing Activities</h2>
+    <p>
+      Per Article 30 of the European Union General Data Protection Act, this report
+      documents Open Humans personal data processing activities in writing. Because
+      our platform enables users to create new personal data inputs and outputs, we
+      maintain a semi-automated report based on the current active projects.
+    </p>
+    <h3>Name & contact details</h3>      
+    <p>
+      <b>Data controller</b> is the Open Humans Foundation, which manages the Open
+      Humans platform,
```

Comment:
Minor nit here, you use commas up here, e.g. "platform,", but later on use colons, for instance on line 54.

Personally, I like colons better :)